example/ios/CameraRollExample.xcodeproj/project.pbxproj

```diff
@@ -657,6 +658,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = 5RFPY33TKL;
```

Comment:
do we need the `development team` additions in this file?

Reply:
I don't think we do, Removing